pkg/utils/tekton/pipelines.go

```diff
@@ -156,3 +161,52 @@ func GetFailedPipelineRunLogs(c *common.SuiteController, pipelineRun *v1beta1.Pi
 	}
 	return failMessage
 }
+
+func GetFailedPipelineRunYaml(pipelineRun *v1beta1.PipelineRun) ([]byte, error) {
+	return yaml.Marshal(pipelineRun)
+}
+
+// StorePipelineRunLogs stores logs and parsed yamls of pipelineRuns into directory of given testName under ARTIFACT_DIR env.
+// In case the files can't be stored in ARTIFACT_DIR, they will be recorder in GinkgoWriter.
+func StorePipelineRun(pipelineRun *v1beta1.PipelineRun, testName string, suiteController *common.SuiteController) error {
+	artifactDir := os.Getenv("ARTIFACT_DIR")
```

Comment:
We need to define by default $PWD/tmp thhe folder artifact if ARTIFACT_DIR env dont exists. You can use: https://github.com/redhat-appstudio/e2e-tests/blob/main/pkg/utils/util.go#L35-L41

Reply:
Thanks! Fixed it.